virtcontainers/types/bridges.go

```diff
@@ -41,9 +47,11 @@ func NewBridge(bt Type, id string, address map[uint32]string) Bridge {
 	var maxCapacity uint32
 	switch bt {
 	case PCI:
-		maxCapacity = pciBridgeMaxCapacity
+		maxCapacity = PCIBridgeMaxCapacity
```

Comment:
`fallthrough` ?

Reply:
yeah, i should change it to `fallthrough`